docs/05-command-line.md

```diff
@@ -21,6 +21,7 @@ $ npx ava --help
     --color                 Force color output
     --no-color              Disable color output
     --reset-cache           Reset AVA's compilation cache and exit
+    --node-arguments        Configure Node.js arguments used to launch worker processes
```

Comment:
We should include sample usage before, given that you need to provide the arguments as a single string.